README.md

```diff
@@ -131,3 +134,24 @@ head(as.data.table(filter), 3)
 final outcome using the supplied measure. Any learner can be passed to
 this filter with `classif.rpart` being the default. Of course, also
 regression learners can be passed if the task is of type “regr”.
+
+### Filter-based Feature Selection
+
+In many cases filtering is only one step in the modeling pipeline. To
+select features based on filter values, one can use
+[`PipeOpFilter`](https://mlr3pipelines.mlr-org.com/reference/mlr_pipeops_filter.html)
+from [mlr3pipelines](https://github.com/mlr-org/mlr3pipelines).
```

Comment:
One sentence per line please :) (maybe worth checking your editor settings, it seems line breaks are enforced at 80 chrs?)

Reply:
Note that this is the md file generated from the Rmd